contracts/Staking.sol

```diff
@@ -35,7 +36,10 @@ contract Staking {
         _;
     }
 
-    constructor() {}
+    constructor(uint32 minNumValidators, uint32 maxNumValidators) {
```

Comment:
What happens when you deploy this contract without specifying these values, will they take default values (`0`)?

Can we confirm this with a unit test?

cc @Kourin1996

Reply:
You cannot deploy a contract without providing the constructor values. You can verify this by trying not to provide them in the tests.
They're here for testing purposes only, as we don't use the constructor in the predeploy process.

Reply:
What's the default value?

Reply:
Yes, in solidity the default uint32's value is zero.

Reply:
Firstly, they are contract fields and must be initialized regardless of the difference between normal deploy and pre-deploy. The deployed bytecode may not have initialization of these fields.

Whan you deploy contract in normal way, you can't deploy contract without required arguments. Then you don't have to care about this.

@zivkovicmilos @PtitLuca

Reply:
@Aleksao998 you have fixed this in the polygon-edge pr correct? @zivkovicmilos please review

Reply:
@lazartravica As @Kourin1996 said when deploying a contract usually these attributes are required. 

When we are pre-deploying with deployed bytecode, we have to set these fields manually using a storage map. We have an e2e test that checks does our default boundaries work. I have tried setting these values as